### PR TITLE
Chapter 17 - Filter by lower bound instead of point estimate

### PR DESCRIPTION
**Pull request by xiaowei-zhang**
Filter positive net value regions by 95% CI lower bound, instead of the point estimate, gives better results in test set.

The outcome needs to be updated as well:

- 18 regions to invest
- Average Net Income: 15.57

chapter17

### CODE REVIEW

File: causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb [hunk 1]
```diff
@@ -661,11 +661,16 @@
    ],
    "source": [
     "# extract the lower bound of the 95% CI from the plot above\n",
-    "regions_to_net = train.groupby(\"region\")[\"net_value\"].mean().to_dict()\n",
+    "regions_to_net = train.groupby('region')['net_value'].agg(['mean', 'count', 'std'])\n",
```
Comment (matheusfacure) [inline causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb:664]: Please alter the text above to reflect this change. Other than that, good to go!

**Review by matheusfacure — CHANGES_REQUESTED** [review]

File: causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb (hunk 1 above)
Comment (xiaowei-zhang) [reply causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb:664]: Which part of the text needs to be changed?

Comment (matheusfacure) [reply causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb:664]: Right above this cell:
```
"To construct this policy, we will do something very simple. If the expected (average) net value of a region is positive, we will do business with it. Technically, \\\\(\\hat{E}[NetValue|Region] > 0\\\\).\n",
    " \n",
    "The following code builds a dictionary where the key is the region and the value is the expected value. Then, the dictionary generator filters only those regions where the expected net value is positive. The result is the regions we will do business with according to our policy."
```
the value is no longer the expected value, but the lower bound of the 95% C.I. right?

Comment (xiaowei-zhang) [reply causal-inference-for-the-brave-and-true/17-Predictive-Models-101.ipynb:662]: Right! I added a new version of text and region list, but did not update the plots. The new title for the hist plots should be "Average Net Income: 15.57" and "Region Policy Net Income: 15.57".

**Review by matheusfacure — APPROVED** [review]
Thank! That should do.

**Pull request merged**